iroh-sync/src/store/fs/migrations.rs

```diff
@@ -95,12 +95,23 @@ fn migration_002_namespaces_v2(tx: &WriteTransaction) -> Result<MigrateOutcome>
         namespaces_v2.insert(&id, (raw_kind, &raw_bytes))?;
         entries += 1;
     }
-    tx.delete_table(NAMESPACES_TABLE_V1)?;
     Ok(MigrateOutcome::Execute(entries))
 }
 
+/// Delete the v1 namespaces table.
+fn migration_003_namespaces_delete_v1(tx: &WriteTransaction) -> Result<MigrateOutcome> {
```

Comment:
can you add a comment why this is its own migration? this is quite surprising

Reply:
:+1: added a commit with a comment